client/src/views/Profile.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Container } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import AuthorInfoSection from '../component/AuthorInfoSection';
+import AuthorTimelines from '../component/AuthorTimelines';
+
+const ProfileView = ({ userData }) => (
+  <Container>
+    <div className="ProfileView">
+      <AuthorInfoSection
```

Comment:
This all information can be passed using context. This will save you from writing all the propTypes, setting defaultProps.

Reply:
Totally agree, I could have but chose not to, since the data had to be passed just one layer down through ProfileView. :slightly_smiling_face: 
ProfilePage (Context Provider)
ProfileView (Context Consumer)
AuthorTimelines (Context Consumer)
TimelineList (Common Component, props)
TImelineCard (Common Component, props)